src/main/java/uk/gov/hmcts/ccd/domain/model/std/validator/globalsearch/SearchCriteriaValidator.java

```diff
@@ -1,24 +1,19 @@
-package uk.gov.hmcts.ccd.domain.model.std.validator;
+package uk.gov.hmcts.ccd.domain.model.std.validator.globalsearch;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import org.springframework.util.CollectionUtils;
 import uk.gov.hmcts.ccd.domain.model.search.global.SearchCriteria;
 
 import javax.validation.ConstraintValidator;
 import javax.validation.ConstraintValidatorContext;
 
 public class SearchCriteriaValidator implements ConstraintValidator<ValidSearchCriteria, SearchCriteria> {
 
-    private static final Logger LOG = LoggerFactory.getLogger(SearchCriteriaValidator.class);
-
     @Override
     public boolean isValid(final SearchCriteria criteria, final ConstraintValidatorContext context) {
         if (criteria != null) {
-            try {
-                return criteria.getNonNullFields();
-            } catch (IllegalAccessException e) {
-                LOG.error(e.getMessage());
-            }
+            // rule: At least one jurisdiction or case type must be provided in the search criteria
+            return !(CollectionUtils.isEmpty(criteria.getCcdJurisdictionIds()))
+                || !(CollectionUtils.isEmpty(criteria.getCcdCaseTypeIds()));
```

Comment:
```suggestion
            return !CollectionUtils.isEmpty(criteria.getCcdJurisdictionIds())
                || !CollectionUtils.isEmpty(criteria.getCcdCaseTypeIds());
```
Minor suggestion to remove unnecessary extra brackets.